v2/car.go

```diff
@@ -17,9 +22,9 @@ var (
 		0x02, // uint(2)
 	}
 	// The size of the CAR v2 prefix in 11 bytes, (i.e. 11).
-	PrefixBytesSize = len(PrefixBytes)
+	PrefixBytesSize = uint64(len(PrefixBytes))
 	// Reserved 128 bits space to capture future characteristics of CAR v2 such as order, duplication, etc.
-	EmptyCharacteristics = new(Characteristics)
+	EmptyCharacteristics = Characteristics{0, 0}
```

Comment:
Anyone can construct this with just `Characteristics{}` or `var zero Characteristics`, right? Seems redundant to expose this. You could always have it unexported if you prefer it for your own code.

Reply:
Would it make sense to have it so that we define what empty characteristic means? The rationale is that, we might want default or empty characteristics to have some non-zero defaults, e.g. Ordered, or Dererministic. In which case, having this variable signifies what the characteristics would be if not specified by the user.

Reply:
I'm not sure. I think I'd expect "all zero bits" to be the default forever, but I'm also not sure what kind of use cases we have in mind for characteristics in the future. You could later on change the writer to set some of the bits by default, and I wonder whether you could say that's fully backwards compatible - it could mean that some downstreams would suddenly start writing different CARs by just upgrading the library, which could definitely surprise developers.

Reply:
Good point thanks